File: SPA/FrontEnd/src/app/Models/Region/RegionModel/region-model.ts
import { PuntoTuristico } from '../../Punto/punto';
export class RegionModel {
    nombre: string;
    puntos: PuntoTuristico[];

    constructor(nombre: string, nuevosPuntos: PuntoTuristico[]) {
        this.nombre = nombre;
        this.puntos = nuevosPuntos;
    }
}

File: ObligatorioDA2/Models/RegionModel.cs
using BuisnessLogic.Domain;
using Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Obligatorio.Models
{
    public class RegionModel
    {
        public string Nombre { get; set; }
        public virtual List<PuntoTuristico> Puntos { get; set; }
        public Region ToEntity() => new Region()
        {
            Nombre = this.Nombre,
            Puntos=this.Puntos,
        };
    }
}
